format/bson/bson.jq

```diff
@@ -0,0 +1,11 @@
+def bson_tovalue:
```

Comment:
Not happy with `*_tovalue`, `*_tojson` might seem like a good idea but in jq terminology "json" is the string representation of a json value. Mayb `*_torepr` as in representation?